dev - OpenBOR - joypad fix

Langerz82 opened the pull request:
dev - OpenBOR - joypad fix

Current BUG:
Extra controllers need to enabled through game menus, and cannot use both DPAD and analog controls (one or the other).

Old issues:
In openBORff left analog is not working. - FIXED.
Not sure which keyboard assignment Attack 3 & 4 uses if any. - FIXED.


Related Issues:
https://github.com/EmuELEC/EmuELEC/issues/1080

shantigilbert commented:
Is this good to merge? or what else needs to be done? (its marked as WIP)

shantigilbert commented on packages/sx05re/emulators/openbor/package.mk:
```diff
@@ -42,6 +42,11 @@ makeinstall_target() {
     mkdir -p $INSTALL/usr/config/emuelec/configs/openbor
-	cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/masterff.cfg $INSTALL/usr/config/emuelec/configs/openbor/masterff.cfg
```
Why is this change needed? did you missed the package for openborff? 

https://github.com/EmuELEC/EmuELEC/blob/dev/packages/sx05re/emulators/openbor/openborff/package.mk

shantigilbert commented on packages/sx05re/emulators/openbor/scripts/openbor.sh:
```diff
@@ -23,16 +23,17 @@ SAVES="${CONFIGDIR}/Saves"
 
 # We start the fake keyboard
-gptokeyb openbor &
+gptokeyb -c /emuelec/configs/gptokeyb/${OB}.gptk openbor &
```
Why are 2 identical gptk files used?

Langerz82 commented:
@shantigilbert 
Ooops I missed that openborff seperate package and updated.
Also I see your point only one gptokeyb profile is needed as they are identical.
I updated the PR to reflect the changes.

Langerz82 commented:
I double check that the gptokeyb script copies into storage in an update, and it does so I'm ok with this to be pushed.

shantigilbert commented:
Thank you
